ios/FluentUI/People Picker/AvatarView.swift

```diff
@@ -753,13 +772,11 @@ class OverflowAvatarView: AvatarView {
         super.init(avatarSize: avatarSize, withBorder: false, style: .circle, preferredFallbackImageStyle: .outlined)
 
         var overflowCountString = NumberFormatter.localizedString(from: NSNumber(value: overflowCount), number: .none)
-        overflowCountString = String(format: "AvatarView.OverflowFormat".localized, overflowCountString)
+        overflowCountString = String(overflowCountString)
```

Comment:
If we are intentionally removing the "+" from the overflow, we shouldn't need line 775, right?
I think we can also make overflowCountString let instead of var.